packages/@react-types/checkbox/src/index.d.ts

```diff
@@ -28,11 +28,7 @@ import {
 } from '@react-types/shared';
 import {ReactElement, ReactNode} from 'react';
 
-export interface ToggleProps extends InputBase, Validation, FocusableProps {
-  /**
-   * The label for the element.
-   */
-  children?: ReactNode,
+export interface ToggleStateProps extends InputBase {
```

Comment:
hmmm that's an interesting one, because right now, the aria hook is the thing that prevents input state changes and interactions with isDisabled. To have stately use it would be a bit redundant, but maybe there's an implementation out there where the interaction isn't prevented and we'd need it here. I think it makes more sense to keep it at the moment.